src/pages/NewPlaylist/index.jsx

```diff
@@ -1,11 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
+import "./styles.scss";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
I don't know this very well, but I wonder whether there is a way to put this into the css :thinking:

Reply:
If we want to set a color theme, it's better to leave it I guess